apps/client-asset-sg/src/app/components/language-selector/language-selector.component.ts

```diff
@@ -0,0 +1,61 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { MatButton } from '@angular/material/button';
+import { MatMenu, MatMenuTrigger } from '@angular/material/menu';
+import { NavigationEnd, Router, RouterLink } from '@angular/router';
+import { AnchorComponent, supportedLangs } from '@asset-sg/client-shared';
+import { isNotNull, isTruthy } from '@asset-sg/core';
+import { Lang } from '@asset-sg/shared';
+import { SvgIconComponent } from '@ngneat/svg-icon';
+import { UntilDestroy } from '@ngneat/until-destroy';
+import { LetModule } from '@rx-angular/template/let';
+import { flow, pipe } from 'fp-ts/function';
+import * as O from 'fp-ts/Option';
+import queryString from 'query-string';
+import { debounceTime, filter, map, startWith } from 'rxjs';
+
+@UntilDestroy()
```

Comment:
This doesn't do anything here as the component doesn't use any manually managed subscriptions.